app/models/flyover_comments/comment.rb

```diff
@@ -1,6 +1,9 @@
 module FlyoverComments
   class Comment < ActiveRecord::Base
     include FlyoverComments::LinkParsing
+
+    scope :with_includes, -> { includes(:children) }
```

Comment:
I think we probably don't want this by default? Some comment threads (esp. on GJOpen) get really long and we load the children (other than the first one) remotely. I think this would load all of them and just not render most?